resources/version.go

```diff
@@ -47,17 +47,20 @@ type ContainerVolumeMount struct {
 // Version as defined by Brizo.
 type Version struct {
 	database.Model
-	UUID          string      `gorm:"not null;unique_index:uix_versions_uuid" sql:"type:varchar(36)" json:"uuid"`
-	Name          string      `gorm:"not null;unique_index:uix_versions_name_environment_id" json:"name"`
-	Slug          string      `gorm:"not null" json:"slug"`
-	Replicas      int         `gorm:"not null" sql:"DEFAULT:'0'" json:"replicas"`
-	EnvironmentID uint        `gorm:"not null;unique_index:uix_versions_name_environment_id" json:"environment_id"`
-	Environment   Environment `gorm:"not null" json:"environment"`
-	Volumes       []Volume    `gorm:"-" json:"volumes"`
-	Containers    []Container `gorm:"-" json:"containers"`
-	Spec          string      `gorm:"type:json" json:"-"`
+	UUID            string       `gorm:"not null;unique_index:uix_versions_uuid" sql:"type:varchar(36)" json:"uuid"`
+	Name            string       `gorm:"not null" json:"name"`
```

Comment:
Needs the `unique_index:uix_versions_name_application_uuid` listed w/in the field tag.